doc/contributing/docs/style.rst

```diff
@@ -174,13 +174,38 @@ Spelling and punctuation
 US vs British spelling
 ~~~~~~~~~~~~~~~~~~~~~~
 
-We use the US English spelling.
+Use the US English spelling.
 
 Check your spelling and punctuation
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Consider checking spelling, grammar, and punctuation with special tools.
 
+Dashes
+~~~~~~
+
+There is no need to insert special symbols like dashes, quotation marks, or apostrophes.
+The documentation :doc:`builder </contributing/docs/build/>`
+renders specific character sequences in the source into correct typographic characters. 
+
+Type three hyphens (``---``) to insert an em dash (---)
+and two hyphens (``--``) for an en dash (--).
+The following rules apply:
+
+*   The shorter **en dash** ``--`` is used to mark ranges and with the meaning "to"
+    in phrases like "Dover--Calais crossing".
```

Comment:
We hardly ever have such cases in our docs, but we can have number ranges, like 4–16 gigabytes.